app/controllers/admin/emergency_messages_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::EmergencyMessagesController < Admin::ApplicationController
+
+  # GET /courts/1/edit
+  def edit
+    @em = EmergencyMessage.find(params[:id])
+  end
+
+  def update
+    @em = EmergencyMessage.find(params[:id])
+
+    respond_to do |format|
+      if @em.update_attributes(params[:emergency_message])
+        purge_all_pages
```

Comment:
what I suggest is to at least stop using it and we can create a separate clean up task to remove it from the code base. What do you think?